include/mrdocs/Metadata/Javadoc.hpp

```diff
@@ -709,6 +711,50 @@ struct Throws : Paragraph
     }
 };
 
+struct Precondition : Paragraph
+{
+    static constexpr Kind static_kind = Kind::precondition;
+
+    Precondition(
+        Paragraph details_ = Paragraph())
+        : Paragraph(
+            Kind::precondition,
+            std::move(details_.children))
+    {
+    }
+
+    bool operator==(const Precondition&)
+        const noexcept = default;
+
+    bool equals(const Node& other) const noexcept override
+    {
+        return kind == other.kind &&
```

Comment:
should this be
```
return static_kind = other.kind &&
```
?

Reply:
It _could_ used `static_kind`, but since every other javadoc node uses `kind` I'm not going to use `static_kind` for just these two. At some point we can just change all the nodes to use `static_kind`.